src/types/ApolloContext.ts

```diff
@@ -0,0 +1,14 @@
+import { Response } from 'express';
+import { Pool, ModuleThread } from 'threads';
+import { ProjectResolverWorker } from '../workers/projectsResolverWorker';
+import { JwtVerifiedUser } from '../services/authorizationServices';
+
+export interface ContextRequest {
+  user: JwtVerifiedUser;
+  auth: any;
```

Comment:
Should we avoid using any? I think we can specify it like this.
```
export interface AuthContextRequest {
   token?: string;
   error?: any;
}
```
The error is kind of hard to define..... as it depends 100% on which graphql function is called with what message.